R/covr.R

```diff
@@ -380,20 +380,22 @@ package_coverage <- function(path = ".",
   # read tracing files
   trace_files <- list.files(path = tmp_lib, pattern = "^covr_trace_[^/]+$", full.names = TRUE)
   coverage <- merge_coverage(trace_files)
+
+  trace_files <- list.files(path = tmp_lib, pattern = "^covr_branch_trace_[^/]+$", full.names = TRUE)
+  branches <- merge_coverage(trace_files)
+
   if (!uses_icc()) {
     res <- run_gcov(pkg$path, quiet = quiet)
+    res_b <- run_gcov(pkg$path, quiet = quiet, gcov_br = TRUE)
```

Comment:
I think you want to streamline it: so far you have `br`, `branches`, `b` to indicate some associations with bramches.

Reply:
We should also think about some logic to globally set whether branches are used not